be/src/exec/olap_table_sink.cpp

```diff
@@ -640,14 +640,25 @@ void OlapTableSink::_convert_batch(RuntimeState* state, RowBatch* input_batch, R
 #else
                     state->append_error_msg_to_file("", ss.str());
 #endif
-                    continue;
+                    exist_null_value_for_not_null_col = true;
+                    break;
                 }
             }
             void* slot = dst_tuple->get_slot(slot_desc->tuple_offset());
             RawValue::write(src_val, slot, slot_desc->type(), _output_batch->tuple_data_pool());
         }
+
+        if (!exist_null_value_for_not_null_col) {
+            if (output_batch->add_row() != RowBatch::INVALID_ROW_INDEX) {
+                commit_rows++;
+                output_batch->get_row(i)->set_tuple(0, dst_tuple);
```

Comment:
If one row is skipped, `get_row(i)` will get a wrong result. I think you should what you get is add_row()'s return value